tests/__snapshots__/multiple-routes.test.js.snap

```diff
@@ -771,8 +924,11 @@ SCRIPTS: [
   "/runtime.js",
   "/5.js",
   "/main.js",
+  "/handlers-Home.js",
```

Comment:
For reference, these were removed in a lockfile update PR a few months ago due to [a bug](https://github.com/gregberge/loadable-components/issues/999) in one of the loadbable deps. This bug was recently fixed, hence why the chunk is back.